### PR TITLE
css-box-model - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
css-box-model - 1

### CODE REVIEW

File: css-box-model/css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+.language {
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
```
Comment (shawnkost) [inline css-box-model/css-box-model/styles.css:12]: You're missing a `padding-top` & `padding-bottom`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Roland!

**Comment by restropiajr** [comment]
css-box-model - 1

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**